proposals/0019-graphql.md

```diff
@@ -0,0 +1,130 @@
+
+## Maturity Justification
+
+We are proposing this package at the "Incubation" level of maturity. I've listed the requirements for this maturity level and responses below:
+
+Minimum Requirements:
+
+* General
+  * Has relevance to Swift on Server specifically: **Argued in Motivation section**
+  * Publicly accessible source managed by an SCM such as github.com or similar: **Satisfied: Repository is stored on github.com**
+    * Prefer to use `main` as the default branch name, in line with [Swift's guidelines](https://forums.swift.org/t/moving-default-branch-to-main/38515): **Satisfied**
+  * Adopt the [Swift Code of Conduct](https://swift.org/community/#code-of-conduct): **Satisfied**
+* Ecosystem
+  * Uses SwiftPM: **Satisfied**
+  * Integrated with critical SSWG ecosystem building blocks, e.g., Logging and Metrics APIs, SwiftNIO for IO: **Satisfied**
+* Longevity
+  * Must be from a team that has more than one public repository (or similar indication of experience): **Satisfied: See the [GraphQLSwift Organization](https://github.com/GraphQLSwift)**
+  * SSWG should have access / authorization to graduated repositories in case of emergency: **This can be arranged**
+  * Adopt the [SSWG Security Best Practices](../security/README.md)): **Satisfied**
+* Testing, CI and Release
+  * Have unit tests for Linux: **Satisfied**
+  * CI setup, including testing PRs and the main branch: **Satisfied**
+  * Follow semantic versioning, with at least one published pre-release (e.g. 0.1.0, 1.0.0-beta.1) or release (e.g. 1.0.0): **Satisfied**
+* Licensing
+  * Apache 2, MIT, or BSD (Apache 2 recommended): **Satisfied: MIT**
+* Conventions and Style
+  * Adopt [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/)
+  * Follow [SSWG Technical Best Practices](#technical-best-practices) when applicable.
+  * Prefer to adopt code formatting tools and integrate them into the CI: **No code formatting tools are integrated currently**
+
+Incubating Requirements:
+
+* Document that it is being used successfully in production by at least two independent end users which, in the SSWG judgement, are of adequate quality and scope.
+  * **Jay Herron () is using it in a commercial product at [PassiveLogic](https://passivelogic.com/). This package is also used in multiple actively maintained open source projects, including [Graphiti](https://github.com/GraphQLSwift/Graphiti), and [Pioneer](https://github.com/d-exclaimation/pioneer).**
```

Comment:
empty `()` can be likely removed?

Reply:
Whoops, meant to link my user account. I've added that link.